integration/flink/flink2/README.md

```diff
@@ -1,49 +1,3 @@
-# Flink Native Lineage Listener
```

Comment:
Did you put that into docs?

Reply:
That's the old description from the times when flink 2 support was a separate jar. 
One or two more PRs and I am planning to rewrite Flink integration documentation which is terribly outdated at the moment.